servant/servant.cabal

```diff
@@ -121,8 +121,8 @@ library
                   , ConstraintKinds
                   , DataKinds
                   , DeriveDataTypeable
+                  , DeriveFunctor
```

Comment:
Why did this need to get turned on?

Reply:
I don't think there is really any system to the extensions enabled. I think it would be best to fix the failing query string test on master first. But I do think this looks sensible. Is TypeFamilyDependencies added for the same reason?

Reply:
Yes. This one is explicitly enabled for `servant/src/Servant/API/ResponseHeaders.hs`.